sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/parsers/full-time-e-cert-record-parser.ts

```diff
@@ -0,0 +1,74 @@
+import { ECertRecordParser } from ".";
+
+/**
+ * Parses a full-time e-Cert record information.
+ */
+export class FullTimeCertRecordParser extends ECertRecordParser {
+  /**
+   * List of e-Cert awards. All federal grants may potentially be listed
+   * while only total provincial grants (BCSG) will be available.
+   */
+  private readonly awards: Record<string, number> = {};
+
+  /**
+   * Initializes a new full-time parsed object.
+   * @param record record to be parsed.
+   */
+  constructor(private readonly record: string) {
+    super();
+    for (let i = 615; i < 715; i += 10) {
```

Comment:
mm, I didn't get the logic of these values, Are 615 and 715, index for awards in the file?

Reply:
ya, I think i got it, they are indexes

Reply:
Yes, they are. I did not add constants or comments because the entire class has indexes all around.
I can add one comment on the top stating it maybe.

![image](https://github.com/bcgov/SIMS/assets/61259237/155348dc-5cf7-4909-95ea-e29a5a901c71)

Reply:
Comment added to the class.